scripts/utils/sign_and_send.js

```diff
@@ -1,17 +1,13 @@
 module.exports = function (web3 = web3, artifacts = artifacts) {
-  const assert = require("assert")
   const axios = require("axios")
-
-  const { signHashWithPrivateKey } = require("../utils/internals")(web3, artifacts)
   const { ZERO_ADDRESS } = require("./constants")
+  const { getSafeCompatibleSignature } = require("./internals")
```

Comment:
`internals.js` requires `(web3, artifacts)` to be specified, otherwise you'd get
```
TypeError: getSafeCompatibleSignature is not a function
```
when running the script.

```suggestion

  const { getSafeCompatibleSignature } = require("./internals")(web3, artifacts)
  const { ZERO_ADDRESS } = require("./constants")
```
I also added a newline for consistency with the imports in other files, it separates internal imports to our project from external imports, like `BN`.